README.md

```diff
@@ -978,121 +977,84 @@ callback         | (Function)  | Callback function executed as a result. The err
 
 APIs available in `require('sfcc').slas`:
 
-`tenant.add(tenantId, shortcode, description, merchantName, contact, emailAddress, fileName)`
+`tenant.add({shortcode, tenant, file})`
```

Comment:
These many argument style interfaces are error prone and very hard to change in a backwards compatible way. 

I've moved all the interfaces to accepting objects with named attributes.

Reply:
I understand. Please see the dicusssion for the original intent and the suggestion in last line of combining best of both worlds